### ISSUE TITLE
Importing passable into a TypeScript file causes compilation errors.

### ISSUE THREAD

**Issue by quietopus** [fiverr/passable#283]
Using `passable@7.5.0`.

Importing `passable` at the top of a `.ts` file as follows:

```javascript
import passable, {enforce} from 'passable';
```

causes TypeScript to generate the following errors upon compilation:

```
node_modules/passable/index.d.ts:1:9 - error TS2614: Module '"passable"' has no exported member 'IEnforceInstance'. Did you mean to use 'import IEnforceInstance from "passable"' instead?

1 import {IEnforceInstance} from "passable";
          ~~~~~~~~~~~~~~~~

node_modules/passable/index.d.ts:13:5 - error TS2666: Exports and export assignments are not permitted in module augmentations.

13     export default passable;
       ~~~~~~

node_modules/passable/index.d.ts:14:5 - error TS2666: Exports and export assignments are not permitted in module augmentations.

14     export { enforce, Enforce, validate, WARN, FAIL, VERSION };
```

No errors are displayed in my IDE (Visual Studio Code) while coding and TypeScript-based auto-completion for `passable` seems to work. The errors only appear upon compilation.

When I remove the `import {IEnforceInstance} from "passable";` statement from `index.d.ts` in the `passable` project root, all compilation errors seem to be resolved.

My `tsconfig.json` is as follows:

```json
{
    "compilerOptions": {
        "allowJs"         : true,
        "lib"             : ["ES2017", "DOM"],
        "target"          : "ES2017",
        "module"          : "CommonJS",
        "outDir"          : "dist",
        "rootDir"         : "src",
        "sourceMap"       : true,
        "incremental"     : true,
        "esModuleInterop" : true
    },
    "include": ["src/**/*"]
}
```

**Comment by Attrash-Islam** [comment]
Thanks for reporting.. Will take a look on this

**Comment by Attrash-Islam** [comment]
Fixed in #284 🎉 Waiting for review

**Comment by Attrash-Islam** [comment]
Resolved in 7.5.1
